transformer_engine/pytorch/attention.py

```diff
@@ -1248,6 +1249,25 @@ def run_iteratively(q, k, v):
 
     return qkv_layout, q, k, v
 
+def check_set_window_size(
+        attn_mask_type: str,
+        window_size_in: Tuple[int, int] = None,
+        window_size_out: Tuple[int, int] = None,
```

Comment:
You are already returning it from the function so why having it as an argument as well?

Reply:
Because if not I'd have to set `window_size_out` to `None` (`use before assignment` error) and then it will return `None`, but in fact it may not be `None`. The usage for this function is also sometimes `(..., window_size, window_size)` and sometimes `(..., window_size, self.window_size)`.

Reply:
Simplified it per slack discussion.